### PR TITLE
The Basics tutorial: Part 2.

### PR DESCRIPTION
**Pull request by ryzhyk**

### CODE REVIEW

File: docs/tutorials/basics/part2.md [hunk 1]
```diff
@@ -1,8 +1,210 @@
...
+description of this technique.
+
+## Takeaways
+
+Let us review what we have learned in this part of the tutorial:
+
+- Feldera operates on **changes**: a Feldera pipeline transforms a stream of
+  input changes to SQL tables into a stream of output changes to SQL views.
+
+- One way to send input changes to a pipelien and consume output changes from it
```
Comment (blp) [inline docs/tutorials/basics/part2.md:205]:
```suggestion
- One way to send input changes to a pipeline and consume output changes from it
```

**Review by blp — APPROVED** [review]

File: docs/tutorials/basics/part1.md [hunk 2]
```diff
@@ -183,9 +183,15 @@ Feldera queries are always-on: the query engine continuously updates
...
 
-> **_NOTE:_** The Web Console currently does not support deleting records.  Use
+> **_NOTE:_** The Web Console does not yet support deleting records.  Use
 > the REST API described in the next part of the tutorial instead.
 
+## Step 6. Stop the pipeline
+
+Click the stop icon <icon icon="bx:stop-circle" /> to shutdown the pipeline.
+
+> **_NOTE:_** All pipeline state will be lost.
```
[the remark this answers is not in the document]
Comment (ryzhyk) [reply docs/tutorials/basics/part1.md:193]: Oh, that's good to know.

File: docs/tutorials/basics/part2.md [hunk 3]
```diff
@@ -1,8 +1,210 @@
...
+
+![Pipeline UUID](pipeline-uuid.png)
+
+## Step 2. Subscribe to output changes
+
+Subscribe to changes to the `PREFERRED_VENDOR` view (here and below, replace the pipeline UUID
+component of the URL with your UUID):
+
+```bash
+curl -X 'POST' 'http://localhost:8080/v0/pipelines/018a0ea6-9165-7e55-95f4-e5d865eeb272/egress/PREFERRED_VENDOR?format=json'
```
[the remark this answers is not in the document]
Comment (ryzhyk) [reply docs/tutorials/basics/part2.md:36]: > The reader is going to have to replace uuids in a lot of places to follow along. Would it be better to use shell variable substitution? Then they could type UUID=<uuid> once and after that just cut-and-paste your examples verbatim.

good idea!

[the remark this answers is not in the document]
Comment (ryzhyk) [reply docs/tutorials/basics/part2.md:36]: > Would it work to pipe the output through jq to make it easier to read?

I found a combination of switches to make it work!

**Pull request merged**